admin/webappsec-charter-2021.html

```diff
@@ -555,6 +556,7 @@ <h3>
 	  <h2>Success Criteria</h2>
 	  <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
 	  <p>Each specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+  	  <p>All new features should be supported by at least two intents to implement before being incorporated in the specification.</p>
```

Comment:
So far as I can tell, only Mozilla and Google have a common practice of sending something like "intent to implement/prototype" emails. WHATWG's [policy document says](https://whatwg.org/workstream-policy#living-standard) "Living Standards are intended to include only those features that are likely to be implemented in major browser engines and are suitable for adoption and implementation by other browser engine developers and integrators.", HTML's PR template includes "At least two implementers are interested (and none opposed):", and their [working mode document has some nuanced suggestions](https://whatwg.org/working-mode#changes).

With that in mind, it might be reasonable to frame this a little less narrowly?

Reply:
Came here to say what @mikewest already said.

Reply:
Would you say instead: "New features should be expected to have at least two implementations, as judged by expressions of interest (and lack of opposition)."?

Another question we've heard is whether features should be dropped after some time window if expected implementations don't materialize. Thoughts?

Reply:
Wendy: yes and yes. 😊 (Though that window should be somewhat large, on the order of years.)

Reply:
I agree with the comments from Mike and Anne. The proposed text in the PR matches the spirit of what we intend in shorthand, but might be too blunt and invite rule-lawyering. I like the nuance of the WHATWG working-mode document and would like to capture more of it. 

I imagine it's not allowed to have external definitions in a charter, since they technically could change out from under us. And it's kind of a lot of text to incorporate wholesale (and might need tweaks besides). As a short form the sentence Wendy proposed two comments above is better.